### PR TITLE
dracut: don't explicitly remount /sysroot rw

### PR DESCRIPTION
**Pull request by crawford**
This feels a bit weird doing the remount in this module, but I don't really have any better ideas.

### CODE REVIEW

File: dracut/10usr-generator/usr-generator [hunk 1]
```diff
@@ -32,6 +32,10 @@ usr=$(cmdline_arg usr)
 usrfstype=$(cmdline_arg usrfstype auto)
 usrflags=$(cmdline_arg usrflags ro)
 
+requires_dir="${UNIT_DIR}/initrd-root-fs.target.requires"
```
Comment (marineam) [inline dracut/10usr-generator/usr-generator]: Since this is pretty static seems like it should just be in module-setup.sh instead.

**Comment by marineam** [comment]
Also, it may be possible to override the default of ro or a kernel command line that includes ro with a fstab but it's been like two months since I last read that code so I don't specifically recall the behavior.

**Comment by crawford** [comment]
Depends on https://github.com/coreos/coreos-overlay/pull/1250.

**Comment by marineam** [comment]
LGTM

**Pull request merged**